pyanaconda/modules/storage/bootloader/systemd.py

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (C) 2022 Arm
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from pyanaconda.modules.storage.bootloader.base import BootLoader, BootLoaderError
+from pyanaconda.core import util
+from pyanaconda.core.configuration.anaconda import conf
+from pyanaconda.core.i18n import _
+from pyanaconda.core.path import join_paths
+from pyanaconda.product import productName
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+__all__ = ["SystemdBoot"]
+
+class SystemdBoot(BootLoader):
+    """Systemd-boot.
+
+    Systemd-boot is dead simple, it basically provides a second boot menu
+    and injects the kernel parms into an efi stubbed kernel, which in turn
+    (optionally) loads it's initrd. As such there aren't any filesystem or
+    drivers to worry about as everything needed is provided by UEFI. Even the
+    console remains on the UEFI framebuffer, or serial as selected by UEFI.
+    Basically rather than trying to be another mini-os (like grub) and duplicate
+    much of what UEFI provides, it simply utilizes the existing services.
+
+    Further, while we could keep stage1 (ESP) and stage2 (/boot) seperate it
+    simplifies things just to merge them and place the kernel/initrd on the
+    ESP. This requires a larger than normal ESP, but for now we assume that
+    this linux installer is creating the partitions, so where the space
+    is allocated doesn't matter.
+
+    """
+    name = "SYSTEMD"
```

Comment:
Is this supposed to be "SYSTMED" or "SDBOOT" ?

Reply:
It should be SDBOOT.

Reply:
Fixed.